packages/framework/src/Model/Product/ProductVisibilityRepository.php

```diff
@@ -111,16 +109,20 @@ protected function refreshGlobalProductVisibility($onlyMarkedProducts)
      */
     public function createAndRefreshProductVisibilitiesForPricingGroup(PricingGroup $pricingGroup, $domainId)
     {
-        $query = $this->em->createNativeQuery(
+        $this->em->getConnection()->executeStatement(
             'INSERT INTO product_visibilities (product_id, pricing_group_id, domain_id, visible)
             SELECT id, :pricingGroupId, :domainId, :calculatedVisibility FROM products',
-            new ResultSetMapping()
+            [
+                'pricingGroupId' => $pricingGroup->getId(),
+                'domainId' => $domainId,
+                'calculatedVisibility' => false,
```

Comment:
Is it necessary to pass static value by parameter?

Reply:
Good point. fixed in 706c46209